.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,42 @@
+// For format details, see https://aka.ms/devcontainer.json. For config options, see the
+// README at: https://github.com/devcontainers/templates/tree/main/src/go
+{
+    "name": "Go Dev Container",
+    "image": "mcr.microsoft.com/devcontainers/go:1-1.23-bookworm",
+    // Features to add to the dev container. More info: https://containers.dev/features.
+    "features": {
+        "ghcr.io/devcontainers/features/go:1": {
+            "version": "latest"
```

Comment:
Is using the latest go version intentional (instead of pinning the same version as in `go.mod`) ?

Reply:
It was intended to use the latest, as we want to update regularly.
I can use the same as go.mod if you insist.